File: ARC/ARC022_A.java
import java.util.Scanner;

public class ARC022_A {
    static Scanner sc=new Scanner(System.in);
    
    public static void main(String[] args){
        String s=sc.next().toUpperCase();
        int a=0;
        boolean f=true;
        if((a=s.indexOf("I",0)) == -1)f=false;
        if((a=s.indexOf("C",a)) == -1)f=false;
        if((a=s.indexOf("T",a)) == -1)f=false;
        System.out.println(f? "YES" : "NO");
    }
}

File: ARC/ARC022_A.py
S = input()

I = False
C = False
T = False

for i in list(S):
    # Tを探す
    if I and C:
        if i == 'T' or i == 't': T = True
    # Cを探す
    elif I:
        if i == 'C' or i == 'c': C = True
    # Iを探す
    else:
        if i == 'I' or i == 'i': I = True

# 全部存在すればYES
if I and C and T:
    print('YES')
else:
    print('NO')

